yt_idefix/_io/vtk_io.py

```diff
@@ -217,7 +219,10 @@ def read_field_offset_index(
         s = line.decode()
         datatype, varname, dtype = s.split()
 
-        if upper_case_varnames:
+        # some versions of Pluto define field names in lower case
+        # so we normalize standard output field names to upper case
+        # to avoid duplicating data in PlutoFields.known_other_fields
+        if varname.upper() in dict(field_info.known_other_fields).keys():
```

Comment:
just noticed this. I'd prefer we use `more_itertools` for these kinds of loops and avoid conversions.
One possible implementation would be
```suggestion
        if varname.upper() in unzip(field_info.known_other_fields)[0]:
```
(requires`from more_itertools import unzip`)
`more_itertools` is currently not listed as a hard dependency to `yt_idefix` (but it *is* already a dependency in `yt` itself), so it would need to be added to `project.dependencies` in `pyproject.toml`.

It also strikes me that this list is all that we need from the `field_info` argument, so maybe it should be the argument instead ?

Reply:
Thanks! First time to know it.

> It also strikes me that this list is all that we need from the `field_info` argument, so maybe it should be the argument instead ?

It sounds reasonable. I'll change it.